mitmproxy/optmanager.py

```diff
@@ -45,33 +92,37 @@ class OptManager(metaclass=_DefaultsMeta):
         Optmanager always returns a deep copy of options to ensure that
         mutation doesn't change the option state inadvertently.
     """
-    _initialized = False
-    attributes = []
-
-    def __new__(cls, *args, **kwargs):
-        # Initialize instance._opts before __init__ is called.
-        # This allows us to call super().__init__() last, which then sets
-        # ._initialized = True as the final operation.
-        instance = super().__new__(cls)
-        instance.__dict__["_opts"] = {}
-        return instance
-
     def __init__(self):
+        self.__dict__["_options"] = {}
         self.__dict__["changed"] = blinker.Signal()
         self.__dict__["errored"] = blinker.Signal()
-        self.__dict__["_initialized"] = True
+        self.__dict__["_processed"] = {}
```

Comment:
I'd love for this to just go away. In any other case than core, we can hang processed data off an addon instance. In the case of body_size_limit, my feeling is that _processed is marginally  nicer than having the core of mitmproxy depend on human-friendly value parsing in multiple places. 

In the longer term we should a) move as much as we can into addons, and b) get rid of this particular instance by passing it up and down our chain as needed. Let's punt on this right at the moment.